### ISSUE TITLE
Breadcrumbs - Update breadcrumbs liquid filter to support lang attribute

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/vets-design-system-documentation#2869]
## Description
This should be done after #2868. We have an open [PR in content build](https://github.com/department-of-veterans-affairs/content-build/pull/2095) where an accessibility issue was surfaced regarding the `lang` attribute. Ticket #2868 was done to add support for the `lang` attribute in `va-breadcrumbs`. This ticket should update the above pull request so it can be merged. 

## Considerations
- Work done in this ticket should use the branch associated with [this pull request](https://github.com/department-of-veterans-affairs/content-build/pull/2095)
- [Original comment highlighting the issue](https://github.com/department-of-veterans-affairs/content-build/pull/2095#issuecomment-2127464252)
- [Slack conversation around the issue](https://dsva.slack.com/archives/C06V7AAFVH7/p1716569568866569)
- lang values are determined through [this existing liquid filter](https://github.com/department-of-veterans-affairs/content-build/blob/b8bbae9f972de759f5eac261cdeff2c4d3b1afd2/src/site/filters/liquid.js#L717-L718)

## Tasks
- [ ] Update the drupal liquid filter and/or the breadcrumb liquid template so `lang` is passed in as a prop.  

## Acceptance Criteria
- [ ] Above tasks have been completed
- [ ] Sitewide team has verified the issue has been resolved in the above pull request

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2869?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by Andrew565** [comment]
This will carry over due to accessibility feedback needing to be incorporated into the breadcrumbs component, which means that the liquid filter updates will not be reviewable until the breadcrumbs component updates are released.

**Comment by Andrew565** [comment]
This will carry over again due to not being able to get the updated component-library release out soon enough for the sitewide reviewers to have due time to review the changes before the end of the sprint.
